stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -31,6 +33,9 @@ public object ChatUI {
     public var style: ChatStyle = ChatStyle()
     public var navigator: ChatNavigator = ChatNavigator()
     public var urlSigner: UrlSigner = UrlSigner.DefaultUrlSigner()
+    public var imageRequestHeaders: Map<String, String> by Delegates.observable(emptyMap()) { _, _, newState ->
+        StreamImageLoader.instance().imageRequestHeaders = newState
+    }
```

Comment:
We can probably avoid storing this in ChatUI altogether like this:

```suggestion
    public var imageRequestHeaders: Map<String, String>
        get() = StreamImageLoader.instance().imageRequestHeaders
        set(value) {
            StreamImageLoader.instance().imageRequestHeaders = value
        }
```